python/mach_bootstrap.py

```diff
@@ -111,7 +110,7 @@ def _activate_virtualenv(topdir):
         except (subprocess.CalledProcessError, OSError):
             sys.exit("Python virtualenv failed to execute properly.")
 
-    execfile(activate_path, dict(__file__=quote(activate_path)))
+    execfile(activate_path, dict(__file__=activate_path))
```

Comment:
@larsbergstrom If that is the only concern, that is great! :smiley: Because by merging this change we are not trading anything. The original code had one of two possible effects:
- did nothing when no quoting was required
- caused virtualenv not to activate when quoting was deemed necessary (on any platform)

Especially it did not help with the "space in the path" problem in any way. I've went into great lengths to better explain it in the issue #10595 and in the commit message.